### PR TITLE
[docs] Added KESL documentation for Security software settings doc

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Added KESL documentation for Security software settings doc.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Added KESL documentation for Security software settings doc.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/stage'...

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 1]
```diff
@@ -6,6 +6,27 @@ lang: ru
...
+- `Network Threat protection (ID: 17)`.
+- `Web Control (ID: 26)`.
+
+{% alert level="info" %}
+Список задач может отличаться в будущих версиях KESL.
+{% endalert %}
+
+Убедитесь, что узлы Kubernetes соответствуют минимальным требованиям к ресурсам, указанным для [DKP](https://deckhouse.ru/products/kubernetes-platform/guides/production.html#требования-к-ресурсам) и [KESL](https://support.kaspersky.com/KES4Linux/12.1.0/ru-RU/197642.htm).
+
+Для совместной работы KESL и Deckhouse может потребоваться оптимизация производительности согласно [рекомендациям Kaspersky](https://support.kaspersky.com/KES4Linux/12.1.0/ru-RU/206054.htm).
```
Comment (mortelumina98) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:32]:
```suggestion
При совместной эксплуатации KESL и Deckhouse может потребоваться оптимизация производительности согласно [рекомендациям Kaspersky](https://support.kaspersky.com/KES4Linux/12.1.0/ru-RU/206054.htm).
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md [hunk 2]
```diff
@@ -5,6 +5,27 @@ permalink: en/security_software_setup.html
 
 If Kubernetes cluster nodes are analyzed by security scanners (antivirus tools), you may need to configure them to avoid false positives.
 
-Deckhouse uses the following directories when working ([download their list in csv...](deckhouse-directories.csv)):
+Deckhouse Kubernetes Platform (DKP) uses the following directories during operation ([download in CSV](deckhouse-directories.csv)):
```
Comment (kirkonru) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md:8]:
```suggestion
Deckhouse Kubernetes Platform (DKP) uses the following directories when running ([download in CSV](deckhouse-directories.csv)):
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md [hunk 3]
```diff
@@ -5,6 +5,27 @@ permalink: en/security_software_setup.html
...
 
 {% include security_software_setup.liquid %}
+
+## Security Software
+
+### KESL
+
+The following are recommendations for configuring Kaspersky Endpoint Security for Linux (KESL) to ensure proper operation with the Deckhouse Kubernetes Platform, regardless of the selected edition.
```
Comment (kirkonru) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md:16]:
```suggestion
The following are recommendations for configuring Kaspersky Endpoint Security for Linux (KESL) to ensure that it operates smoothly with Deckhouse Kubernetes Platform (whatever edition you choose).
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md [hunk 4]
```diff
@@ -5,6 +5,27 @@ permalink: en/security_software_setup.html
...
+
+To ensure compatibility with DKP, the following tasks must be disabled on the KESL side:
+
+- `Firewall_Management (ID: 12)`.
+- `Web Threat Protection (ID: 14)`.
+- `Network Threat Protection (ID: 17)`.
+- `Web Control (ID: 26)`.
+
+{% alert level="info" %}
+The list of tasks may differ in future versions of KESL.
```
Comment (kirkonru) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md:26]:
```suggestion
Note that the task list may be different in future KESL versions.
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md [hunk 5]
```diff
@@ -5,6 +5,27 @@ permalink: en/security_software_setup.html
...
+{% endalert %}
+
+Ensure that your Kubernetes nodes meet the minimum resource requirements specified for [DKP](https://deckhouse.io/products/kubernetes-platform/guides/production.html#resource-requirements) and [KESL](https://support.kaspersky.com/KES4Linux/12.1.0/en-US/197642.htm).
+
+For the combined use of KESL and Deckhouse, performance optimization may be required according to [Kaspersky's recommendations](https://support.kaspersky.com/KES4Linux/12.1.0/en-US/206054.htm).
```
Comment (kirkonru) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP.md:31]:
```suggestion
If KESL and Deckhouse are run together, you may be required to do some performance tuning as per [Kaspersky recommendations](https://support.kaspersky.com/KES4Linux/12.1.0/en-US/206054.htm).
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10840](https://github.com/deckhouse/deckhouse/pull/10840) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10841](https://github.com/deckhouse/deckhouse/pull/10841) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Pull request merged**